pub-matic-htb.js

```diff
@@ -259,9 +367,7 @@ function PubMaticHtb(configs) {
                  * is usually some sort of placements or inventory codes. Please replace the someCriteria
                  * key to a key that represents the placement in the configuration and in the bid responses.
                  */
-
-                /* ----------- Fill this out to find a matching bid for the current parcel ------------- */
-                if (curReturnParcel.xSlotRef.someCriteria === bids[i].someCriteria) {
+                if (bids[i].impid === curReturnParcel.xSlotRef.bid_id) {
```

Comment:
Please ensure a null/undefined check for bids exist. If the bids are undefined, please treat them as pass.